contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -94,6 +94,14 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     uint8 linkPremiumPercentage
   );
 
+  event FallbackWeiPerUnitLinkUsed(
+    int256 fallbackWeiPerUnitLink,
+    int256 weiPerUniLink,
+    uint256 stalenessSeconds,
+    uint256 blockTimestamp,
```

Comment:
`blockTimestamp`used here could be also extracted from the transaction receipt - are we keeping it here just for completeness so we can compare it with `timestamp` more easily? Also, `stalenessSeconds` is something that you could easily extract via `getConfig`, but perhaps it's safer to leave it because this parameter may change during the coordinator's lifetime.

Reply:
Got it, let's keep all of that in that case, since this event should not be emitted too often.